examples/platforms/nrf52840/usb-cdc-uart.c

```diff
@@ -210,6 +213,7 @@ static void processConnection(void)
             app_usbd_start();
         }
     }
+    otSysEventSignalPending();
```

Comment:
This call shouldn't be needed here.

`mConnected` and `mReadyToStart` variables are modified from USB handler `usbdUserEventHandler`.